src/dmd/clone.d

```diff
@@ -521,12 +521,12 @@ FuncDeclaration buildOpEquals(StructDeclaration sd, Scope* sc)
 
 /******************************************
  * Build __xopEquals for TypeInfo_Struct
- *      static bool __xopEquals(ref const S p, ref const S q)
+ *      static bool __xopEquals(ref const S q, ref const S p)
  *      {
  *          return p == q;
```

Comment:
I think it would make more sense to swap the return expression to `q == p`, as q is the "this" parameter, same as buildXopCmp.

Reply:
Yeah shouldn't matter, all to be replaced by your #13590 anyway.